src/App.tsx

```diff
@@ -6,7 +6,7 @@ function App() {
   return (
   <Router>
         <Main />
-  </Router>
+  </Router>    
```

Comment:
Could you remove the extra spaces after </Router> so there is not change to this file?

Reply:
done